Automattic/wp-calypso
# Gutenberg: Add likes and sharing sidebar

#### Changes proposed in this Pull Request

This PR adds a Gutenberg sidebar extension that allows changing the settings to enable/disable Likes and enable/disable sharing buttons on a per-post basis.

#### Testing Steps

1. Double-check [sharing button prerequisites](https://github.com/Automattic/wp-calypso/pull/29744#issuecomment-459443781).
1. Load a new or existing post in Gutenberg
1. Under the Jetpack sidebar,  there should be a Likes and Shares section with checkboxes to toggle these options:

<img width="299" alt="screen shot 2019-01-24 at 3 44 01 pm" src="https://user-images.githubusercontent.com/52152/51707327-e8355e80-1fee-11e9-9e90-4a915f255728.png">

5. Make sure you can set/unset them and that these settings are appropriately reflected on the post (you can check via the API console or the classic editor, or by just viewing the post on the front-end).

6. Also test by disabling Likes globally and then toggling them ON on a per-post basis.


See #28332 #28528 D20639-code D24043-code
